lib/animina/accounts/resources/photo.ex

```diff
@@ -175,4 +195,75 @@ defmodule Animina.Accounts.Photo do
       reference :user, on_delete: :delete
     end
   end
+
+  def create_optimized_photos(record) do
+    case check_if_image_magick_is_installed() do
+      {:ok, _} ->
+        resize_images_with_imagemagick(record)
```

Comment:
Please change the function name to `resize_image`. Singular and without the imagemagick part. We might change imagemagick to something else in the future but would keep the function name.

Reply:
That is clear , done